clients/web/test/specRunner.js

```diff
@@ -176,6 +176,16 @@ page.onLoadFinished = function (status) {
         page.evaluate(function () {
             if (window.girderTest) {
                 girderTest.promise.done(function () {
+                    // Allow Jasmine to compare RegExp using toEqual, toHaveBeenCalledWith, etc.
+                    jasmine.getEnv().addEqualityTester(function (a, b) {
```

Comment:
Hm, I see the value here, but this could cause some crazy headache in the future if someone is trying to actually test that a method was called with a certain regexp using jasmine mocks.

Is there a way to just add a new `toHaveBeenCalledWithRegex` method so that we don't have to change comparisons globally?